README.md

```diff
@@ -10,6 +10,8 @@ The contents of this repository are licensed under the [Apache License 2.0](./LI
 
 To get involved, please see [Code of Conduct](https://github.com/eiffel-community/.github/blob/master/CODE_OF_CONDUCT.md) and [contribution guidelines](https://github.com/eiffel-community/.github/blob/master/CONTRIBUTING.md).
 
+Technical Committee meeting agenda and minutes are kept in [MEETINGS.md](./MEETINGS.md).
```

Comment:
Maybe it would fit better to put this information under 'Contact' below, within a sub-heading called 'Eiffel Community Technical Committee', where we could add some more info about the coming TC later on?

Reply:
that makes sense. let me amend the PR.